<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="N110630" failures="1" tests="2" name="LoginTest" time="17.096" errors="0" timestamp="21 Apr 2021 22:32:24 GMT">
  <testcase classname="tests.LoginTests" name="invalidLoginTest_InvalidUserNameInvalidPassword" time="7.679">
    <failure type="java.lang.AssertionError" message="expected [Hatalı kullanıcı adı veya şifre.] but found [auth microservisindeki undefined geçici olarak erişimini kapalı. Lütfen daha sonra Tekrar deneyiniz! Hata kodu: 500]">
      <![CDATA[java.lang.AssertionError: expected [Hatalı kullanıcı adı veya şifre.] but found [auth microservisindeki undefined geçici olarak erişimini kapalı. Lütfen daha sonra Tekrar deneyiniz! Hata kodu: 500]
	at pages.BasePage.assertEquals(BasePage.java:56)
	at pages.LoginPage.verifyLoginUserName(LoginPage.java:43)
	at tests.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTests.java:23)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- invalidLoginTest_InvalidUserNameInvalidPassword -->
  <testcase classname="tests.LoginTests" name="invalidLoginTest_EmptyUserEmptyPassword" time="5.517"/>
</testsuite> <!-- LoginTest -->
